packages/babel-traverse/src/path/index.js

```diff
@@ -151,17 +150,17 @@ export default class NodePath {
   }
 }
 
-assign(NodePath.prototype, require("./ancestry"));
-assign(NodePath.prototype, require("./inference"));
-assign(NodePath.prototype, require("./replacement"));
-assign(NodePath.prototype, require("./evaluation"));
-assign(NodePath.prototype, require("./conversion"));
-assign(NodePath.prototype, require("./introspection"));
-assign(NodePath.prototype, require("./context"));
-assign(NodePath.prototype, require("./removal"));
-assign(NodePath.prototype, require("./modification"));
-assign(NodePath.prototype, require("./family"));
-assign(NodePath.prototype, require("./comments"));
+Object.assign(NodePath.prototype, require("./ancestry"));
```

Comment:
Can merge this into a single call:

```js
Object.assign(NodePath.prototype,
  require("./ancestry"),
  require("./inference"),
  require("./replacement"),
  require("./evaluation"),
  require("./conversion"),
  require("./introspection"),
  require("./context"),
  require("./removal"),
  require("./modification"),
  require("./family"),
  require("./comments")
);
```

(Something similar will land in https://github.com/babel/babel/pull/5351)